src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/KestrelHttpParser.cs

```diff
@@ -70,7 +70,7 @@ public KestrelHttpParser(IKestrelTrace log)
         private unsafe void ParseRequestLine<T>(T handler, byte* data, int length) where T : IHttpRequestLineHandler
         {
             int offset;
-            Span<byte> customMethod;
+            Span<byte> customMethod = default(Span<byte>);
```

Comment:
That's because it might not be initialised when you use it (as its only initalized in `if`).

Still should be `var customMethod = default(Span<byte>);`

moar `var`